content/docs/started/k8s/kfctl-k8s-istio.md

```diff
@@ -12,10 +12,17 @@ This config creates a vanilla deployment of Kubeflow with all its core component
 
 Follow these steps to deploy Kubeflow:
 
-1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
+1. Download the latest `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
 
     ```
```

Comment:
Thanks for this update! I'd like to give people the option of doing the download and unpack manually if they prefer.

Suggestion: Leave the original instructions in place but offer your option to do the download and unpack in just 2 commands.

Reply:
Thanks for the review Sarah! 
Sorry for the late reply

Reply:
Does this look good?

Reply:
Thanks!